internal/resources/fabric/service_token/models.go

```diff
@@ -508,9 +519,31 @@ func networkTerraformToGo(networkList []interface{}) fabricv4.SimplifiedTokenNet
 	var network fabricv4.SimplifiedTokenNetwork
 	networkListMap := networkList[0].(map[string]interface{})
 	uuid := networkListMap["uuid"].(string)
+	href := networkListMap["href"].(string)
 	type_ := networkListMap["type"].(string)
-	network.SetUuid(uuid)
-	network.SetType(fabricv4.SimplifiedTokenNetworkType(type_))
+	name := networkListMap["name"].(string)
+	scope := networkListMap["scope"].(string)
+	locationList := networkListMap["location"].(*schema.Set).List()
+
+	if uuid != "" {
```

Comment:
These checks are good, but they should follow the required/optional flow of the schema. If a field is required, you don't have to check for empty strings because Terraform will guarantee it has a value by throwing an error message to the user if they attempt to do a plan or apply without it.